### DISCUSSION TITLE
LFX Mentorship Streaming Jobs for Service Dependencies Potential Project Interest

### DISCUSSION THREAD

**Discussion by mohamedawnallah** [jaegertracing/jaeger#4589] [General]
I'm genuinely interested in this potential LFX mentorship project. Currently, I have identified two repositories that might be related to this [project proposal](https://github.com/jaegertracing/jaeger/issues/4457#issuecomment-1570362166). They are as follows:

1) [Jaeger Analytics Flink](https://github.com/jaegertracing/jaeger-analytics-flink)
2) [Spark Dependencies](https://github.com/jaegertracing/spark-dependencies)

I'm slightly confused because, in the first repository "Jaeger Analytics Flink," I noticed service dependency streaming jobs in Apache Flink. Could you please clarify if this is related to the project proposal [in case of using Apache Flink], and if that's the case, what improvements does this project address?

**Comment by yurishkuro** [comment]
I booked a ticket https://github.com/jaegertracing/jaeger/issues/4590, please ask there for clarifications.

**Comment by mohamedawnallah** [comment]
Thanks i'm gonna close the discussion here
